packages/ui/examples/E2_FetchWithStrapiRequest.tsx

```diff
@@ -1,17 +1,42 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { useEffect, useState } from 'react'
 
-import { Box } from '@chakra-ui/react'
+import { Box, Text } from '@chakra-ui/react'
 
 import { strapiRequest } from '@fc/services/common/strapiRequest'
+import { Blog } from '@fc/types'
 
 export const FetchWithStrapiRequest = () => {
-  const [blogs, setBlogs] = useState([])
+  const [blogs, setBlogs] = useState<Blog[]>([])
 
   useEffect(() => {
     // TODO: fetch blogs with strapiRequest by using the API_URL
     // Remember that fetcher is a wrapper around axios that adds the token and api url to the request
+
+    const fetchBlogs = async () => {
+      try {
+        const res = await strapiRequest<Blog>({
+          endpoint: 'blogs',
+          page: 1,
+          pageSize: 10,
+          sort: ['createdAt:desc'],
+          filters: {},
+        })
+
+        setBlogs(res.data)
+
+        console.log(res.data)
+      } catch (error) {
+        console.log(error)
+      }
+    }
+
+    fetchBlogs()
   }, [])
 
-  return <Box>{/* TODO: Show only title of the blogs */}</Box>
+  return <Box>{/* TODO: Show only title of the blogs */}
+  {blogs.map((blog:Blog)=>(
```

Comment:
Blog type is redundant here. And again, it's not yet formatted with prettier.

```suggestion
  {blogs.map((blog)=>(
```